### DISCUSSION TITLE
IZAR+D1 Mini+CC1101

### DISCUSSION THREAD

**Discussion by zibous** [wmbusmeters/wmbusmeters#835] [Ideas]
I did some experiments with wmbusmeters, DVT SDR_STICK, NANOCUL and ended up with: Wemos D1 Min, CUL + CC1101 (esphome).

![d1Mini868Mhz](https://user-images.githubusercontent.com/30198737/216345483-a1dc2d8f-a743-4304-9bcc-a4b9a40efccd.png)

- Water meter with IZAR module (Diehl IZAR RC 868 I R4 PL), IZAR Radio Compact Hall is designed for mobile reading and fixed network remote reading of Diehl Metering meters.
- [Wemos D1 Mini, CUL + CC1101](https://amzn.eu/d/0WQv07I)
- [Fayme CC1101 868MHZ Funk ÜBertragung Antennen Transceiver Modul, Grün](https://amzn.eu/d/i5YwBkR)

`Wemos D1 Mini, CUL + CC1101` (esphome) is the best solution for me because the resource requirement is low.

see: https://github.com/zibous/ha-watermeter/blob/master/esphome/README.md
        https://github.com/zibous/ha-watermeter/blob/master/esphome/wmbus-minid1.yaml

What I still miss

Only the total_m3 are currently decoded from the telegram. I am still looking for support on how to determine the further information from the telegram:

    Alarm (current_alarms, previous_alarms)
    Water last month (last_month_total_m3)
    Last Month date (last_month_measure_date)
    Battery (remaining_battery_life_y)
    Periode (transmit_period_s)

I don't know how to include the necessary libraries or whether there is a 
simple solution for decoding the additional information ?

**Comment by zibous** [comment]
Solved by Szczepan Leon, perfect work.

see: https://github.com/SzczepanLeon/esphome-components

**Comment by SzczepanLeon** [comment]
@zibous 
Can I use on my page that beautiful ESP+CC picture?

**Reply by zibous** [reply]
Yes but i have a better one..

https://github.com/zibous/ha-watermeter/blob/master/esphome/watermeter_d1mini.md
